tools/cli/services/create.js

```diff
@@ -1,25 +1,70 @@
-const fs = require('fs');
-const execute = require('../utils/execute');
-const { red } = require('../utils/colors');
-const { join } = require('path');
+const fs = require("fs");
+const execute = require("../utils/execute");
+const { red } = require("../utils/colors");
+const { join } = require("path");
+
+function runNpmInstallInDirectoryAndExitIfFailed(directoryName) {
+  try {
+    execute(
+      `npm install --prefix ${directoryName}`,
+      "npm install dependencies"
```

Comment:
This should be the expiation of this command, not the actual command. For example: "installing dependencies".